extensions/packageManager/src/node/index.ts

```diff
@@ -249,30 +132,30 @@ export default async (composer: IExtensionRegistration): Promise<void> => {
     },
     getFeed: async function (req, res) {
       // why an array? In the future it is feasible we would want to mix several feeds together...
-      const packageSources = [req.query.url];
+      const packageSources: IPackageSource[] = [req.query];
 
-      let combined = [];
-      for (const url of packageSources) {
+      let combined: IPackageDefinition[] = [];
+      for (const packageSource of packageSources) {
         try {
-          let raw;
-          if (fs.existsSync(url)) {
-            const rawlocal = await crawlLocalFeed(url);
-            // cast this to the form of the http response from nuget
-            raw = { data: { data: rawlocal } };
-          } else {
-            raw = await axios.get(url);
-          }
-          const feed = await normalizeFeed(raw.data);
-          if (Array.isArray(feed)) {
-            combined = combined.concat(feed);
+          const feed: IFeed = await new FeedFactory(composer).build(packageSource);
+          const packageQuery: IPackageQuery = {
```

Comment:
this blocks the use of the default query, so it undoes the narrower default query.
this should only be used if a default query is not specified.

Reply:
Yep, changed to this logic. Added a todo since in reality we want the provided query to further filter with the package query as basis. However we don't need that right now.

Reply:
Pushing a fix shortly.

Reply:
thanks!